stm/stm.ml

```diff
@@ -65,7 +65,7 @@ end
 
 (* During interactive use we cache more states so that Undoing is fast *)
 let interactive () =
-  if !Flags.ide_slave || !Flags.print_emacs || not !Flags.batch_mode then `Yes
+  if !Flags.ide_slave || not !Flags.batch_mode then `Yes
```

Comment:
Well, `coqtop -emacs -compile` works and changes the way warnings are displayed (with tags rather than colors)... perhaps it should error instead? (It seems not very useful without the `<prompt>` tags, at least for applications I'm imagining, like the bug-minimizer.)